src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -448,7 +453,7 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
 
             try
             {
-                await ExecuteWithWatchersAsync(instance, parameterHelper, trace, logger, functionCancellationTokenSource);
+                await ExecuteWithWatchersAsync(instance, parameterHelper, trace, logger, functionCancellationTokenSource, _jobHost, _config);
```

Comment:
Yeah, let's avoid refactoring all this to make it instance, which will result in a bunch of cascading changes. Hamza is just following the existing pattern here.